### DISCUSSION TITLE
Steam release?

### DISCUSSION THREAD

**Discussion by ShayBox** [LibreSprite/LibreSprite#375] [General]
Has a Steam release ever been considered? I realize it's not Libre software but realistically not everyone runs exclusively libre software, having a non-portable self-updating way to install LibreSprite would be nice, similar to Aseprite, Blender, Godot, etc. (It would also give more visibility)  
I would recommend winget but as of now that doesn't support installing portable zips, only running installers, so that's off the table.

**Comment by pegvin** [comment]
The devs are working on a rewrite named dotto, so i guess they'll invest the time and money on that.
